ext/lib/mgmt/mcumgr/ext/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(base64)
```

Comment:
an ext inside an ext, this is rather unlucky placement, if we are going to import this, it should be imported on its own as another library inside Zephyr's ext/ and made available to other components.